src/Redirectr.Web/appsettings.json

```diff
@@ -6,8 +6,5 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*",
-  "Redirectr": {
-    "ShortUrlPath": "s"

```

Comment:
Shouldn't we add a default then inside `RedirectrOptions`?

Reply:
Good catch! thanks